packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -516,6 +532,8 @@ class _CupertinoDatePickerDateTimeState extends State<CupertinoDatePicker> {
         return 0;
       case CupertinoDatePickerMode.date:
         break;
+      case CupertinoDatePickerMode.monthYear:
+        break;
```

Comment:
Nit: Do we no longer need these `break` statements in Dart 3?

Reply:
It doesn't throw errors without them. They probably aren't necessary since they are at the end of the switch and would continue to the following assert with or without the breaks. I don't see anything in the Dart documentation for switch statements indicating that they would be needed, so I will remove them.

Reply:
It seemed as though many tests were failing without a break. It is possible that it was the merge, but I added one break to the end and it seems to have resolved most of the failing tests. Seems odd to require a break at the end of switch, but this is really good to know!